src/node/cli.ts

```diff
@@ -264,6 +265,13 @@ export const options: Options<Required<UserProvidedArgs>> = {
     `,
     deprecated: true,
   },
+  "lng": {
+    type: "string",
+    description: `
+      Language show on login page, more infomations to read up on
+      https://en.wikipedia.org/wiki/IETF_language_tag.
```

Comment:
```suggestion
      Language to show on the login page, more info see 
      https://en.wikipedia.org/wiki/IETF_language_tag
```

Reply:
I had put the description to the `--locale` flag.